docs/en/about-appium/platform-support.md

```diff
@@ -63,6 +66,24 @@ Please refer to these driver docs for setup instructions.
 * Support for automating vendor-provided or third-party apps: Yes
 * Support for automating custom, non-standard UI controls: No
 
+### macOS Support
+
+macOS automation is supported with below drivers.
+
+* The [Mac2Driver](/docs/en/drivers/mac2.md) is for macOS 10.15 or later
+* The (deprecated) [MacDriver](/docs/en/drivers/mac.md) is for lower macOS versions
+
 ### Windows Desktop Support
 
-See the [Windows Driver](/docs/en/drivers/windows.md) doc for details
+Windows automation is supported with below drivers.
+
+* The [WinAppDriver](/docs/en/drivers/windows.md)
+* The [geckodriver](/docs/en/drivers/gecko.md) for Firefox
+
+### You.i Engine Support
```

Comment:
I think it would make sense to mention which of these drivers are supported by Appium team and which of them are supported by vendors/community

Reply:
yea, i agree with it. I wrote this style since no good naming came up then, but vendors/community seems good 👍